packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -11,7 +11,11 @@ on:
       - master
       - main
 
-permissions: read-all
+permissions:
+  contents: read
+{{#if_eq orgName "grafana"}}
+  id-token: write
+{{/if_eq}}
```

Comment:
Are we sure we want to introduce Grafana specific code to create-plugin?

Reply:
I am kind of leaning towards what Jack says, and try to keep this more generic (not introducing internal plugin development related knowledge in the templates).

Reply:
I guess I can see your worries. It is easy it becomes a slippery slope with lots of Grafana internal specific logic added to the template. On the other hand, we are creating lots of pluggins internally so it might be worth having some of this stuff added via the template. I'm a bit torn 🙈

Reply:
so what is the alternative? Another separate workflow for internal devs?

Reply:
Yep my biggest concern here isn't so much this single addition but that we open the door for a community tool to become peppered with our own internal teams "needs". I would vote that we create separate workflows for our internal teams. From what I've gathered they each have their own ideas and would be good to pool those folk together to create something that they can share internally. I'm pretty sure most of them edit/delete this workflow anyhow.

Additionally this template variable is based on user input. If I want my plugin to be called `grafana-awesome-panel` I might (through trial and error) come to the conclusion that to do that I need to use `grafana` as the org name when I scaffoled. Eventually I submit my plugin for review and I'm asked to change the org name unaware that my plugin was scaffolded with specifics used by Grafanas own plugin teams.